students/tfbanks/Lesson02/gridprinter.py

```diff
@@ -0,0 +1,15 @@
+# This is the result of manual processes moved into function form.  
+# I cleaned up during the process and deleted the old methods so the end result is just the function
+
+
+def grid (x,y): #Defines a function to print grid to specified dimensions
```

Comment:
You can remove the space between the function name and parentheses.  i.e., def grid(x,y):